Sources/LogtoClient/LogtoClient/LogtoClient+AccessToken.swift

```diff
@@ -14,7 +14,12 @@ extension LogtoClient {
     }
 
     func getAccessToken(by refreshToken: String, for resource: String?) async throws -> String {
-        let key = buildAccessTokenKey(for: resource, scopes: [])
+        // Hard code profile scope when requesting token for userinfo
+        // Force pass in offline_access as scope to avoid ID Token being returned
+        let scopes = resource == nil ?
+            [LogtoUtilities.Scope.profile.rawValue] :
```

Comment:
Since we have added `profile` in `generateSignInUri`, we can keep the `scopes` null when no resource is provided, and the `profile` claims will always be inside the idToken.
<img width="1279" alt="image" src="https://user-images.githubusercontent.com/10806653/176879141-1cea7268-3145-4f9d-b9cb-7e51df5aaf06.png">

Reply:
this has been updated in another PR